src/databricks/labs/ucx/source_code/python/python_infer.py

```diff
@@ -66,15 +66,15 @@ def _infer_values(cls, node: NodeNG) -> Iterator[Iterable[NodeNG]]:
             yield from cls._safe_infer_internal(node)
 
     @classmethod
-    def _safe_infer_internal(cls, node: NodeNG):
+    def _safe_infer_internal(cls, node: NodeNG) -> Iterator[Iterable[NodeNG]]:
         try:
             yield from cls._unsafe_infer_internal(node)
-        except InferenceError as e:
-            logger.debug(f"When inferring {node}", exc_info=e)
+        except (InferenceError, KeyError) as e:
```

Comment:
is it enough?

Reply:
I don't know yet, at least it will help us log the node and continue with it being uninferable